packages/odyssey/src/scss/base/_typography-link.scss

```diff
@@ -13,14 +13,14 @@
 // limitations under the License.
 
 a {
-  color: cv('action');
+  color: cv('blue');
```

Comment:
It feels like we should either swap in tokens for these or make the cv() array semantic